rust/cubesql/cubeclient/src/models/v1_cube_meta.rs

```diff
@@ -30,15 +32,15 @@ impl V1CubeMeta {
         measures: Vec<crate::models::V1CubeMetaMeasure>,
         dimensions: Vec<crate::models::V1CubeMetaDimension>,
         segments: Vec<crate::models::V1CubeMetaSegment>,
-        joins: Option<Vec<crate::models::V1CubeMetaJoin>>,
     ) -> V1CubeMeta {
         V1CubeMeta {
             name,
             title: None,
+            description: None,
             measures,
             dimensions,
             segments,
-            joins,
+            joins: None,
```

Comment:
@mcheshkov m? did we remove it?

Reply:
It's not marked as `required` in schema, so `openapi-generator` dropped in from `new`. I don't think we are using `V1CubeMeta::new` anyway.

https://github.com/cube-js/cube/blob/05ea7e25014b7ac72e32fa0391ce3ea99ab3e01f/packages/cubejs-api-gateway/openspec.yml#L125-L129